cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DocumentRevision.java

```diff
@@ -312,6 +312,13 @@ public void initialiseSequence(long sequence) {
         }
     }
 
+    /**
+     * @return Whether the body has been modified
```

Comment:
modified since when?

`DocumentRevision` is api_public so we should try to be clear about what the method does or is for or possibly just say it is for internal use.